cmd/podman/common/completion.go

```diff
@@ -426,6 +426,12 @@ func AutocompleteCreateRun(cmd *cobra.Command, args []string, toComplete string)
 		return nil, cobra.ShellCompDirectiveNoFileComp
 	}
 	if len(args) < 1 {
+		// check if the rootfs flag is set
+		// if it is set to true we have to provide path completion
+		rootfs, err := cmd.Flags().GetBool("rootfs")
+		if err == nil && rootfs {
+			return nil, cobra.ShellCompDirectiveDefault
```

Comment:
Just curious: why not `ShellCompDirectiveFilterDirs` ? Can `rootfs` ever be something other than a directory path? (e.g. tarball, image)?

Reply:
No it has to be a directory.

Reply:
Good catch, I completely forgot that this option exists.